### DISCUSSION TITLE
Merging cutscenes between scenarios? (Re1/ re2)

### DISCUSSION THREAD

**Discussion by SolidFat** [IntelOrca/biorand#679] [Ideas]
I was wondering since RE1 and RE2 have scenario/ character specific cutscene triggers, would there be a chance to cross them between characters and scenarios you're playing on during door randos? Like if you play on Claire's file in RE2 could it be possible to get the kissing cutscene that normally only plays for Leon? Or in RE1 if you're playing as Chris you can get Barry to come save you in the ceiling room? I know you can already force yourself into the opposite scenario, but was curious if in door randos if it can just mishmash all the cutscenes together into one basically regardless of characters or scenarios. Might be something fun if its possible to do.

**Comment by IntelOrca** [comment]
It is not so simple. Each room has two files, one for Leon and another for Claire. They contain different scripts and animations needed for the cutscenes. Biorand would need to make a new version of each room that contains everything from both character's files. So it is possible, but a lot of work. I have decided to prioritse my time on generating new random cutscenes and events which will hopefully be available by the end of the year.

**Reply by SolidFat** [reply]
Ahh that makes total sense. Yeah if the coding structure would need to be totally altered then yeah its probably  not worth it. Looking forward to the new random generator you're working on!
